Character Scale related fixes having to do with rendering - Testing and Other Fun Information.txt

```diff
@@ -0,0 +1,60 @@
+The lower-limit for a character's rigged object's scale is just there for QoL.
+
+
+
+Testing:
+• Tested that the scale = 0 QoL fix does indeed work, and no longer causes a crash.
+
+Note: At the lowest size (a value of 1 in-editor) and nearby sizes, characters are difficult to see from a distance due to being small (though they are still rendered), but getting very very close to the model will cause it to stop being rendered at all.
+I don't really think this is a problem though, considering how impractical it is to have characters that are *that* small anyways, and because a size of 15 (in-editor) is tiny and works perfectly from very very close to very very far range.
+
+At the bottom of this "Testing" section, there's a summary of the range of in-editor values for character scale that have very little to no problems at all. (As far as rendering goes anyways.)
+
+
+• Tested that the "No Breathing" param does indeed stop large characters from doing any breathing anims, and that it also works for smaller characters and doesn't cause any problems for them besides stopping their breathing anims.
+Note: A character's "chest position" will be stuck at wherever it is at the moment when the "No Breathing" param is set to "true", regardless of if it's in a wonk position or not.
+Resetting the level with 'L' will fix this.
+
+
+• Tested that having multiple characters of different sizes fighting/being around each other doesn't cause crashes or behave "unexpectedly" (of course, 100 scale characters fighting a 20000 scale character is gonna be mad jank but it still "works" y'know).
+
+
+• Tested that using the editor to set a character's scale param to the following (in-editor) values causes them to render correctly (no flickering and such) at close range, medium range, and long range:
+15 - 250, 1000, 2000, 5000, 10000, 20000
+
+At a size of 20000 sometimes the model will flicker slightly and infrequently when moving the editor camera *very* close to their body. I'd say that 20000 is the highest that
+one *should* go, and 15 is the lowest.
+
+
+I am gonna note that this really does only (hopefully) fix rendering stuff, very large or small characters are still wonk in most other gameplay ways (ragdolling/ledge-climbing/fighting and such) though that makes sense considering Overgrowth never was built around that. 
+Well... Except for rats having bad ledge-climb stuff, dog attacks sometimes hitting even if their fist/foot isn't close to the player, I think cats and wolves suffer from this issue too.
+I think that the hitbox issue can be solved via the "range_adjust" thingy I put up a while back. (This _does_ kinda come with the issue of it being cumbersome to have copies of each different attack for each different race just so they have fitting range_adjust values, but personally I think it's worth it for the sake of fairness/balance since getting hit by phantom dog paws isn't the most fun.)
```

Comment:
This shouldn't be checked in, unless it's in the commit message or comments/description of the PR.

If the code itself is confusing, there may be value to putting some of this in comments directly with the code it applies to.